modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsSnapshot.java

```diff
@@ -110,9 +110,15 @@
     /** Number of non-{@code null} values in the cache. */
     private int size;
 
+    /** Number of non-{@code null} values in the cache. */
```

Comment:
Maybe to add to comment "as a long type"?

Reply:
Done.